cmd/convox/services.go

```diff
@@ -298,6 +305,32 @@ func cmdServiceInfo(c *cli.Context) error {
 	return nil
 }
 
+func cmdServiceURL(c *cli.Context) error {
+	if len(c.Args()) != 1 {
+		stdcli.Usage(c, "url")
+		return nil
+	}
+
+	name := c.Args()[0]
+
+	service, err := rackClient(c).GetService(name)
+	if err != nil {
+		return stdcli.ExitError(err)
+	}
+
+	if service.Status == "failed" {
+		fmt.Printf("Reason  %s\n", service.StatusReason)
+	}
+
+	if service.URL == "" {
+		fmt.Printf("URL does not exist for %s\n", service.Name)
```

Comment:
Maybe use `stdcli.ExitError` here so we get the `ERROR:` prefix and non-zero exit code. Since this will probably be used in scripts I think the extra information would be helpful.